micromamba/tests/test_package.py

```diff
@@ -148,6 +149,11 @@ def test_extract_compress(cph_test_file: Path, tmp_path: Path):
     assert_sorted(names[: len(info_files)])
 
 
+# Only run this test if zstandard is available.
+@pytest.mark.skipif(
+    not hasattr(conda_package_handling.conda_fmt, "ZSTD_COMPRESS_LEVEL"),
+    reason="zstandard not available (required by conda_package_handling)",
+)
```

Comment:
Is `zstandard` not available in some platforms? I mean why do we need to skip the test?